.github/workflows/convert-to-pdf.yml

```diff
@@ -0,0 +1,32 @@
+name: Wersje tekstowe to PDF
+# This workflow is triggered on pushes to the repository.
+on:
+  push:
+    branches:
+      - main
+    # Paths can be used to only trigger actions when you have edited certain files, such as a file within the /docs directory
+    paths:
+      - 'wersje tekstowe/**.md'
+      - 'wersje tekstowe/images/**'
+
+jobs:
+  converttopdf:
+    name: Build PDF
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: baileyjm02/markdown-to-pdf@v1.2.0
+        with:
+          input_dir: wersje tekstowe
+          output_dir: pdfs
+          images_dir: wersje tekstowe/images
+          # for example <img src="./images/file-name.png">
+          image_import: ./images
+          # Default is true, can set to false to only get PDF files
+          build_html: false
+          build_pdf: true
+      - uses: actions/upload-artifact@v3.1.2
+        with:
+          name: Ustawy o PIT
+          path: pdfs
```

Comment:
```suggestion
          path: 'wersje pdf'
```

Reply:
```suggestion
          path: wersje pdf
```

Reply:
trzeba to tez zmienic bo wyskoczy err

Reply:
tutaj też trzeba to zmienić, inaczej, wyskoczy błąd

Reply:
poprawione